### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/config/theme/layout.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#739] [Announcements]
# https://theme-hope.vuejs.press/zh/config/theme/layout.html

以下选项控制主题布局。

https://theme-hope.vuejs.press/zh/config/theme/layout.html

**Comment by kings1990** [comment]
logo最终展示出来的元素结果缺少 alt 属性

**Reply by Mister-Hope** [reply]
这是根据你上次汇报的alt和旁边文字相同的错误开展的修复。logo已经标记了aria-hidden，也就是无障碍应该忽略它。开个新issue把审查工具的提示贴上吧

**Comment by PhyQxx** [comment]
面包屑导航除了末级用的文件标题名，上级默认用的文件夹名称大驼峰写法，怎么配置成中文

**Reply by Mister-Hope** [reply]
自己建 README 然后设置标题

**Reply by PhyQxx** [reply]
该文件夹根目录新建README.md后，面包屑点击之后会进去空白页面，不再是目录页面了

**Reply by Mister-Hope** [reply]
自己手动放置 `<Catalog />` 组件

https://ecosystem.vuejs.press/zh/plugins/features/catalog.html#catalog-1
